### PR TITLE
Add a --interpret-deps flag.

### PR DESCRIPTION
**Pull request by judah**
This flag lets you specify a subset of dependencies that should all be interpreted.  For example:

```
hrepl TARGET --interpret-deps=//some/project
```
Then every Haskell target living under `//some/project` will be interpreted, as if a separate `-package` flag was passed for it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,31 +61,50 @@ For more information about `hrepl`'s command-line flags, run `hrepl --help`.
 
 ### Compiled Dependencies
 
-You may also load modules from the dependencies of your target(s), using
-`:module` or `import`. For example:
+
+By default, `hrepl` will compile any dependencies of your target(s) with Blaze
```
Comment (jinwoo) [inline README.md]: Blaze -> Bazel ?

File: hrepl/tests/run_tests.sh [hunk 2]
```diff
@@ -13,7 +13,8 @@ set -x
 bazel build "$@"
 
 OUTPUT="$(mktemp -d)"
-trap "bazel --output_base=$OUTPUT/output-base clean --expunge && rm -rf $OUTPUT" EXIT
+echo $OUTPUT
+#trap "bazel --output_base=$OUTPUT/output-base clean --expunge && rm -rf $OUTPUT" EXIT
```
Comment (jinwoo) [inline hrepl/tests/run_tests.sh]: Why is this change needed?

Comment (judah) [reply hrepl/tests/run_tests.sh]: Sorry, it was from debugging; reverted.

File: README.md (hunk 1 above)
Comment (judah) [reply README.md]: Done.
